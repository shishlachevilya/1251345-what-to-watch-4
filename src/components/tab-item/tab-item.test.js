import React from "react";
import renderer from "react-test-renderer";
import TabItem from "./tab-item";

describe(`TabItem component test`, () => {
  it(`component should renders correctly`, () => {
    const tree = renderer
    .create(
        <TabItem
          label={`test`}
          onChangeActiveTab={() => {}}/>
    )
    .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
