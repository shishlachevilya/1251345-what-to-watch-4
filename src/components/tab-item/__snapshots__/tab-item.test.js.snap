// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabItem component test component should renders correctly 1`] = `
<a
  className="movie-nav__link"
  href="#"
  onClick={[Function]}
>
  test
</a>
`;
